elyra/pipeline/processor_kfp.py

```diff
@@ -350,6 +351,17 @@ def export(self, pipeline, pipeline_export_format, pipeline_export_path, overwri
 
         return pipeline_export_path  # Return the input value, not its absolute form
 
+    def _collect_envs(self, operation: Operation, **kwargs) -> Dict:
+        """
+        Amends envs collected from superclass with those pertaining to this subclass
+
+        :return: dictionary containing environment name/value pairs
+        """
+        envs = super()._collect_envs(operation, **kwargs)
+        # Only Unix-style path spec is supported.
+        envs['ELYRA_WRITABLE_CONTAINER_DIR'] = self.WCD
```

Comment:
@akchinSTC is this really only used by kfp and not applicable for airflow?

Reply:
As far as I can tell yes this is only used for KFP for locating a suitable writable location for the mlpipeline metrics stuff that the kfp UI uses. These metrics do not exist in airflow or the airflow processor.

Reply:
Yes